collectors/plugins.d/README.md

```diff
@@ -19,6 +19,8 @@ from external processes, thus allowing Netdata to use **external plugins**.
 |[apps.plugin](/collectors/apps.plugin/README.md)|`C`|linux, freebsd|monitors the whole process tree on Linux and FreeBSD and breaks down system resource usage by **process**, **user** and **user group**.|
 |[charts.d.plugin](/collectors/charts.d.plugin/README.md)|`BASH`|all|a **plugin orchestrator** for data collection modules written in `BASH` v4+.|
 |[cups.plugin](/collectors/cups.plugin/README.md)|`C`|all|monitors **CUPS**|
+|[ebpf.plugin](/collectors/ebpf.plugin/README.md)|`C`|linux|monitors different metrics on environments using kernel internal functions.|
```

Comment:
```suggestion
|[ebpf.plugin](https:/.github.com/netdata/netdata/blob/master/collectors/ebpf.plugin/README.md)|`C`|linux|monitors different metrics on environments using kernel internal functions.|
```

@thiagoftsm  Thanks for the update, IMPORTANT: We are trying to push the guideline; use of GH absolute links in every markdown (https://github.com/netdata/netdata/pull/14344#issue-1561772522)

Reply:
I accepted all suggestions, thanks!